### PR TITLE
all tabbed content to ref server v2.19

### PR DESCRIPTION
**Pull request by rosieyohannan**
Tabbed content was created to show difference between v2.19.x and Cloud but they were just labelled "Server". Now Server 3 is close to parity with Cloud we need to update the tab labels to be more specific :-)

https://circleci.atlassian.net/browse/DOCS-9?atlOrigin=eyJpIjoiOGQxNDhkNWI2ZDI2NDRjOGFiZDI0NzM3MjhmODc5NTEiLCJwIjoiaiJ9

### CODE REVIEW

File: jekyll/_cci2/hello-world.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ In the top left, you will find the Org switcher.
 {:.tab.switcher.Cloud}
 ![Switch Organization Menu]({{ site.baseurl }}/assets/img/docs/org-centric-ui_newui.png)
 
-{:.tab.switcher.Server}
+{:.tab.switcher.Server v2.19}
```
Comment (ryanwohara) [inline jekyll/_cci2/hello-world.md:145]: This one is missing the dash.

Should we standardize? You have v2.19, v2.19.x, and v2 - I just don't want users to be confused.

Comment (rosieyohannan) [reply jekyll/_cci2/hello-world.md:145]: Oh weird I thought I had fixed this, maybe I didn't push it up. Let me check! Thanks

File: jekyll/_cci2/hello-world.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ In the top left, you will find the Org switcher.
 {:.tab.switcher.Cloud}
 ![Switch Organization Menu]({{ site.baseurl }}/assets/img/docs/org-centric-ui_newui.png)
 
-{:.tab.switcher.Server}
+{:.tab.switcher.Serverv2}
```
Comment (rosieyohannan) [inline jekyll/_cci2/hello-world.md:145]:
```suggestion
{:.tab.switcher.Server-v2}
```

File: jekyll/_cci2/getting-started.md [hunk 3]
```diff
@@ -274,7 +274,7 @@ comment %} TODO: Job {% endcomment %}build with the SSH enabled option.
 {:.tab.switcher.Cloud}
 ![Rebuild With SSH]( {{ site.baseurl }}/assets/img/docs/rebuild-with-SSH_newui.png)
 
-{:.tab.switcher.Server}
+{:.tab.switcher.Server-v2.19.x}
```
Comment (rosieyohannan) [inline jekyll/_cci2/getting-started.md:277]:
```suggestion
{:.tab.switcher.Server-v2}
```

**Review by ryanwohara — APPROVED** [review]

**Pull request merged**